app/assets/stylesheets/style.css

```diff
@@ -143,6 +143,10 @@ a .fa-white,
   min-width: 190px;
 }
 
+#header .navbar #searchform_input:focus {
+  box-shadow: 2px 3px rgba(21, 156, 228, 0.4) inset !important;
```

Comment:
Hi @Jordan-Rob, it seems the code you've suggested is targeting the DOM element at the same level with the `id=header` and `class=navbar`. If you could change some bit of that to target an inner element then that should do the trick!!

```diff
- #header .navbar #searchform_input:focus {
+ #header .input-group #searchform_input:focus {
```

also in #9634, the border styling which was provided was
```javascript
  box-shadow: 0 0 0 3px rgba(21, 156, 228, 0.4) inset !important;
```
but your's is 
```javascript
  box-shadow: 2px 3px rgba(21, 156, 228, 0.4) inset !important;
```

which is not achieving the desired box-shadow, so changing that is also necessary...
```diff
- box-shadow: 2px 3px rgba(21, 156, 228, 0.4) inset !important;
+ box-shadow: 0 0 0 3px rgba(21, 156, 228, 0.4) inset !important;
```

Hope this helps! Feel free to ask for any help if required.✌️

Reply:
@Tlazypanda  @waridrox Thanks for the support and feedback let me look into it and rectify it

Reply:
Thanks for pointing that out let me work on it